https://chromiumcodereview.appspot.com/9403024

fix up build warning:

In file included from .../base/memory/raw_scoped_refptr_mismatch_checker.h:11:0,
                 from .../base/task.h:10,
                 from ../shill/cellular.h:11,
                 from cellular.cc:5:
.../base/tuple.h: In instantiation of 'void DispatchToMethod(ObjT*, Method, const Tuple0&)
                                       [with ObjT = shill::Cellular, Method = void (shill::Cellular::*)()]':
.../base/task.h:159:9:   instantiated from 'void ScopedRunnableMethodFactory<T>::RunnableMethod<Method, Params>::Run()
                                            [with Method = void (shill::Cellular::*)(), Params = Tuple0, T = shill::Cellular]'
cellular.cc:410:1:   instantiated from here
.../base/tuple.h:540:13: error: unused parameter 'arg' [-Werror=unused-parameter]
cc1plus: all warnings being treated as errors

--- a/base/tuple.h
+++ b/base/tuple.h
@@ -537,7 +538,7 @@ inline Tuple8<A&, B&, C&, D&, E&, F&, G&
 // Non-Static Dispatchers with no out params.
 
 template <class ObjT, class Method>
-inline void DispatchToMethod(ObjT* obj, Method method, const Tuple0& arg) {
+inline void DispatchToMethod(ObjT* obj, Method method, const Tuple0&) {
   (obj->*method)();
 }
 
